plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetFeedHandler.java

```diff
@@ -45,7 +45,7 @@ public Response handle(@Nonnull final Context context) throws Exception {
 
     final String base = getRepositoryUri(context);
 
-    final String feed = facet.feed(base, operation, context.getRequest().getParameters());
+    final String feed = null; //facet.feed(base, operation, context.getRequest().getParameters());
```

Comment:
This PR includes all of the ongoing CMA work, so yeah, this is half-written NuGet code.